models/search_result_test.go

```diff
@@ -75,6 +75,30 @@ var _ = Describe("SearchResult", func() {
 		})
 	})
 
+	Describe("#FindSearchResultByKeywordId", func() {
+		Context("given the keyword already exist", func() {
```

Comment:
Another small typo and `already` could be omitted:

```suggestion
		Context("given the keyword exists", func() {
```